precompiles/distribution/integration_test.go

```diff
@@ -315,6 +315,8 @@ var _ = Describe("Calling distribution precompile from EOA", func() {
 	// =====================================
 	Describe("Execute queries", func() {
 		It("should get validator distribution info - validatorDistributionInfo query", func() {
+			// FIXME this could be broken
```

Comment:
Whats this comment about?

Reply:
Many interfaces change on the sdk v0.50. This includes the `x/staking` module.
For this case in particular, previously (in v0.47.x), the `GetOperator()` returned a `sdk.ValAddress`. Now, it returns a `string`. This change breaks many parts of the code